lib/runner.js

```diff
@@ -50,12 +50,19 @@ function repositoryToNativePackageName(repository/*:string*/) {
   //
   // matches[1] = "rtfeldman"
   // matches[2] = "node-test-runner"
-  var matches = repository.match(/\/([^\/]+)\/([^\/\.]+)\.git/);
+  var matches = repository.match(/\/([^\/]+)\/([^\/]+)\.git/);
 
   if (matches instanceof Array) {
     var username = matches[1];
     var repoName = matches[2];
 
+    //"A dot in repository name breaks `elm-make`" https://github.com/elm-lang/elm-make/issues/106
+    if (repoName.indexOf('.') >= 0) {
+      console.error("Repository name '" + repoName + "' includes the dot symbol, this can cause issues with 'elm-make'");
+      console.error("Please rename the repository or elm-test will not be able to run the tests.");
```

Comment:
Good error messages are hard. As an alternative, what do you think of something like this?

_Elm currently doesn't support having periods in the `user/project` part of the `repository` field of `elm-package.json`.
Aborting test run._